lib/yard-sorbet/sig_to_yard.rb

```diff
@@ -95,4 +92,16 @@ def self.convert_type(node)
       [node.source]
     end
   end
+
+  def self.build_generic_type(node)
+    return node.source if node.children.empty? || node.type != :aref
+
+    collection_type = node.children.first.source
+
+    member_type = node.children.last.children
+                      .map { |child| build_generic_type(child) }
+                      .join(', ')
```

Comment:
Let's consistently use double-quotes:
```suggestion
                      .join(", ")
```

Reply:
The rubocop rules don't allow double quotes without string interpolation

Reply:
Ugh, ok. Nevermind then.